tests/manager/manager-executeBulkWrite_error-009.phpt

```diff
@@ -2,6 +2,7 @@
 MongoDB\Driver\Manager::executeBulkWrite() with invalid options
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_clean(); ?>
```

Comment:
This shouldn't be necessary, as the test doesn't actually insert anything. There's no risk of a duplicate key exception from the test leaving behind data.

`skip_if_not_live()` would be more appropriate here.

Reply:
I'll note that even though we're testing InvalidArgumentExceptions here, those are thrown from [`phongo_execute_bulk_write()`](https://github.com/mongodb/mongo-php-driver/blob/1.6.0alpha3/php_phongo.c#L715), which is _after_ [server selection](https://github.com/mongodb/mongo-php-driver/blob/1.6.0alpha3/src/MongoDB/Manager.c#L515) in the Manager method.